sysutils/pkg/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        freebsd pkg 1.10.4
+categories          sysutils archivers
+platforms           darwin
+license             BSD
+maintainers         {@apowers313 ato.ms:apowers} \
+                    openmaintainer
+description         Package management tool for FreeBSD
+long_description    Pkg is the Next Generation package management tool for FreeBSD. Its main goals are to faciliate remote binary package upgrades. It also works with ports without remote binary packages.
+homepage            https://wiki.freebsd.org/pkgng
+
+depends_lib         port:libarchive \
+                    port:openssl
+
+checksums           sha256 ced2627aae44b95a752565f2bc067888a6f4f3c2e856c233fc1567016788f6cb \
+                    rmd160 b30e8e926fa44612347bfcd81f1896bb44e48198 \
+                    size 3398284
+
+test.run            yes
+
+pre-configure {
+    if {![file exists "${worksrcpath}/configure"]} {
+        system "cd ${worksrcpath} && ./autogen.sh"
```

Comment:
Please change this to use the -W flag to set the path. It should look something like this:

`system -W ${worksrcpath} autogen.sh`

Reply:
Fixed in latest commit.